docs/pages/core-concepts.mdx

```diff
@@ -214,17 +214,16 @@ authenticate to Teleport via a Single Sign-On (SSO) solution.
 
 See our guide to [Authentication Options](./reference/authentication.mdx).
 
-### Trusted Cluster
-
-A **Trusted Cluster** is a remote **Teleport cluster** joined to your own
-cluster via a trust relationship. Users in your Teleport cluster can access
-resources in a Trusted Cluster.
-
-In a Trusted Cluster relationship, the remote cluster is called a **leaf
-cluster**. Your cluster, if you are adding a leaf cluster to it, is called a
-**root cluster**. 
-
-Read our [Trusted Clusters architecture
-guide](./architecture/trustedclusters.mdx) for how Trusted Clusters work and our
-[how-to guide](./management/admin/trustedclusters.mdx) for how to configure
-Trusted Clusters.
+### Trusted clusters
+
+A **trusted cluster** consists of a **root cluster** and one or more **leaf clusters** 
```

Comment:
Technically "trusted cluster" is singular and doesn't "consist of" multiple clusters.

We often use the term "trusted clusters" to mean "any setup where you have multiple clusters that have some sort of trust relationship between them."

In the singular form though, "trusted cluster" typically refers to a root cluster. For example, if cluster A is a root and cluster B is a leaf, then cluster B has one "trusted cluster" resource pointing at cluster A.

Reply:
> "trusted cluster" typically refers to a root cluster. 
Your explanation makes sense to me in the sense that the leaf trusts the CA of the root. However, I don't think I would say that is the "typical" usage. I've seen it used generically and used to mean the leaf cluster. It seems to be what this paragraph originally said: the "remote cluster" (the leaf) joins "your cluster" (the root). 

Treating "Trusted Cluster" and "Trusted Clusters" as proper nouns encourages ambiguity.

I didn't go through and try to make all of the contextual changes. Still, I think it makes a lot more sense to talk about in terms of a trust relationship between a root cluster and a leaf cluster and do away with the generic references to Trusted Clusters or a trusted cluster that sometimes means the root and sometimes means the leaf.